SDWebImage/UIView+WebCache.m

```diff
@@ -94,6 +91,9 @@ - (void)sd_internalSetImageWithURL:(nullable NSURL *)url
                 });
             }
 #endif
+            if (progressBlock) {
```

Comment:
This order is more suitable for user...At most of time, people can write their business code in progressBlock but not in `SDWebImageIndicator` protocol implementation. So if they need some check of current status, putting the `progressBlock` after that indicator was called, can make a help.